.github/workflows/update-hackage-and-chap.yml

```diff
@@ -0,0 +1,42 @@
+name: Update Hackage and CHaP
+
+on:
+  workflow_dispatch: # allows manual triggering
+  schedule:
+    - cron: '0 0 * * 0' # runs weekly on Sunday at 00:00
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Nix
+        uses: cachix/install-nix-action@v18
+        with:
+          # Use last stable nixos channel and the same nix as in channel:
+          install_url: https://releases.nixos.org/nix/nix-2.11.1/install
+          nix_path: nixpkgs=channel:nixos-22.11
```

Comment:
Just wondering, why not 23.05?

Reply:
That's a good question. I simply compied what we have in the other nix-based workflows ([check-mainnet-config.yml](https://github.com/input-output-hk/cardano-node/blob/master/.github/workflows/check-mainnet-config.yml), [check-nix-config.yml](https://github.com/input-output-hk/cardano-node/blob/master/.github/workflows/check-nix-config.yml), [release-upload.yaml](https://github.com/input-output-hk/cardano-node/blob/master/.github/workflows/release-upload.yaml)) to keep the workflow consistent.

Perhaps we should review them all.